config/routes.rb

```diff
@@ -3,13 +3,13 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   root to: "dashboard#index"
 
-  resources :projects, only: %i[new create]
+  resources :projects, only: %i[new create] do
+    resources :approvals, only: %i[new create], controller: "project_rejections"
+    resources :rejections, only: %i[new create], controller: "project_approvals"
+  end
   resources :my_projects, only: %i[index]
 
-  resources :pending_projects, only: [:index] do
-    resources :project_approvals, only: %i[new create]
-    resources :project_rejections, only: %i[new create]
-  end
```

Comment:
Pulled these out of the pending_projects namespace because it was getting a little bit absurd with the `pending_project_project_...` Happy to change it back but it seemed like a reasonable move at the time.

Reply:
maybe `resources :project_approvals, as: :approvals` ?